certificate-policy.md

```diff
@@ -1784,67 +1784,55 @@ It shall contain the value id-kp-OCSPSigning {1 3 6 1 5 5 7 3 9}.
+CA audits shall be performed by a Qualified Auditor. A Qualified Auditor means a natural person, Legal Entity, or group of natural persons or Legal Entities that collectively possess the following qualifications and skills:
 
-1. Independence from the subject of the audit;
-2. The ability to conduct an audit that addresses the criteria specified in an Eligible Audit Scheme (see Section 8.1);
-3. Employs individuals who have proficiency in examining Public Key Infrastructure technology, information security tools and techniques, information technology and security auditing, and the third-party attestation function;
-4. (For audits conducted in accordance with the WebTrust standard) licensed by WebTrust;
-5. Bound by law, government regulation, or professional code of ethics; and
+1. Independence from the subject of the audit
+2. The ability to conduct an audit that addresses the criteria specified in an Eligible Audit Scheme (see Section 8.4)
+3. Employs individuals who have proficiency in examining Public Key Infrastructure technology, information security tools and techniques, information technology and security auditing, and the third-party attestation function
+4. For audits conducted in accordance with the WebTrust standard, licensed by WebTrust
+5. Bound by law, government regulation, or professional code of ethics, and
 6. Except in the case of an Internal Government Auditing Agency, maintains Professional Liability/Errors & Omissions insurance with policy limits of at least one million US dollars in coverage.
 
 ## 8.3 Assessor's relationship to assessed entity
-The compliance auditor either shall be a private firm that is independent from the entities (CA and RAs) being audited, or it shall be sufficiently organizationally separated from those entities to provide an unbiased, independent evaluation. An example of the latter situation may be an Federal agency Inspector General. To insure independence and objectivity, the compliance auditor may not have served the entity in developing or maintaining the entity’s CA Facility or certification practices statement. The FPKIPA shall determine whether a compliance auditor meets this requirement.
-
-The operating Agency and Management Authority of each CA is responsible for identifying and engaging a qualified auditor.
+The Qualified Auditor either shall be a private firm that is independent from the CA being audited, or it shall be sufficiently organizationally separated from the CA to provide an unbiased, independent evaluation. An example of the latter situation may be an Federal agency Inspector General. To insure independence and objectivity, the Qualified Auditor may not have served the CA in developing or maintaining the CPS. The FPKIPA shall determine whether the Qualified Auditor meets the requirements for independence and objectivity.
```

Comment:
Is a federal IG within the same agency likely to meet root store requirements for independence?

Reply:
Yes, I believe so.  Although unlikely to be used in practice given the small intended size of the infrastructure, the clause was included to align with current FPKI certificate policies.  IGs are independent.  

I'll mark this for review by general counsel.

Reply:
> IGs are independent.

They're independent in many ways, and the intent of Congress and the intent of the IGs is for them to be independent, but from a reporting structure standpoint, they do report to the agency head. This may just be a judgment call for the root stores to make, but if you anticipate this actually being exercised, then I suggest we take the temperature of root store providers on this issue.